authentik/events/models.py

```diff
@@ -160,6 +160,51 @@ def get_events_per(
         return self.get_queryset().get_events_per(time_since, extract, data_points)
 
 
+class EventBatch(models.Model):
```

Comment:
should inherit from the base class `ExpiringModel`

Reply:
Done!